cinema/views.py

```diff
@@ -125,6 +159,14 @@
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
 
+    def get_permissions(self):
+        if self.action in (
+                "list",
+                "create"
+        ):
+            return (IsAdminOrIfAuthenticatedReadOnly(),)
+        return (IsAuthenticated(),)
```

Comment:
The `get_permissions` method in `OrderViewSet` allows 'create' action for authenticated users, which might be intended for users to create their orders. Ensure this aligns with the application's access control policy.